src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -25,6 +25,17 @@ import vendorTransformer from '../transformers/vendor.transformer';
 import sendMailToAdvisor from '../utils/transporter.utils';
 
 export default class ReimbursementRequestService {
+  /**
+   * Returns all reimbursement requests in the database that are created by a user.
+   * @param recipient the user retrieving their reimbursement requests
+   */
+  static async getUserReimbursementRequests(recipient: User): Promise<Reimbursement_Request[]> {
```

Comment:
this should return a shared reimbursement request, you may need to merge megans branch into yours to get some of the shared types